Orange/widgets/unsupervised/owdistancetransformation.py

```diff
@@ -52,6 +52,8 @@ def __init__(self):
 
         gui.auto_commit(self.controlArea, self, "autocommit", "Apply",
                         checkbox_label="Apply on any change")
+        gui.separator(box)
+        box.layout().addWidget(self.report_button)
```

Comment:
@janezd Did you mean `gui.auto_commit(box, ...)`? As long as we agree it is not required to place `report_button` explicitly ...